provider/oci/storage_volumes.go

```diff
@@ -154,11 +163,21 @@ func (v *volumeSource) CreateVolumes(ctx envcontext.ProviderCallContext, params
 	if params == nil {
 		return []storage.CreateVolumesResult{}, nil
 	}
+	var credErr error
```

Comment:
The pattern that I'm using is to define `credError` is updated from `nil` when a problem with the credentials is detected. We can then test that independently of other errors that might be generated upstream.